lib/web_ui/lib/src/engine/html/clip.dart

```diff
@@ -330,15 +390,16 @@ class PersistedPhysicalShape extends PersistedContainerSurface
     if (oldSurface.color != color) {
       _applyColor();
     }
-    if (oldSurface.elevation != elevation ||
+    if (oldSurface.path != path || oldSurface.elevation != elevation ||
```

Comment:
Do we have a test that covers the update codepath? We frequently get bugs that only manifest when the DOM nodes are updated from a previous state.

Reply:
Done.